src/reporters/console/console-reporter.js

```diff
@@ -196,7 +196,7 @@ export default class ConsoleReporter extends BaseReporter {
         (err, answer) => {
           if (err) {
             if (err.message === 'canceled') {
-              process.exit(1);
```

Comment:
Yeah I think you're right. I thought about the same thing and don't know why I didn't merge the `reject` part. Update coming.